ui/app/AppLayouts/Wallet/panels/TokenSelectorPanel.qml

```diff
@@ -0,0 +1,303 @@
+        subitems            [model]  - submodel of collectibles/collections of the group
+            key             [string] - balance
+            name            [string] - name of the subitem
+            balance         [int]    - balance of the subitem
+            icon            [url]    - icon of the subitem
+    **/
+    property alias collectiblesModel: collectiblesSfpm.sourceModel
+
+    signal assetSelected(string key)
+    signal collectionSelected(string key)
+    signal collectibleSelected(string key)
+
+    SortFilterProxyModel {
+        id: assetsSfpm
+
+        filters: SearchFilter {
+            roleName: "name"
+            searchPhrase: assetsSearchBox.text
+        }
+    }
+
+    SortFilterProxyModel {
+        id: collectiblesSfpm
+
+        filters: SearchFilter {
+            roleName: "group"
+            searchPhrase: collectiblesSearchBox.text
+        }
+    }
+
+    component SearchFilter: RegExpFilter {
+        required property string searchPhrase
+
+        pattern: `*${searchPhrase}*`
+        caseSensitivity : Qt.CaseInsensitive
+        syntax: RegExpFilter.Wildcard
+    }
+
+    component Search: SearchBox {
+        input.leftPadding: root.leftPadding
+        input.rightPadding: root.leftPadding
+        minimumHeight: 56
+        maximumHeight: 56
+        input.showBackground: false
+        focus: visible
+    }
+
+    contentItem: ColumnLayout {
+        StatusTabBar {
+            id: tabBar
+
+            StatusTabButton {
+                text: qsTr("Assets")
+                width: implicitWidth
+            }
+
+            StatusTabButton {
+                text: qsTr("Collectibles")
+                width: implicitWidth
+            }
+        }
+
+        StackLayout {
+            Layout.maximumHeight: 400
+
+            currentIndex: tabBar.currentIndex
+
+            ColumnLayout {
+                Layout.preferredHeight: visible ? implicitHeight : 0
+                spacing: 0
+
+                Search {
+                    id: assetsSearchBox
+
+                    Layout.fillWidth: true
+                    placeholderText: qsTr("Search assets")
+                }
+
+                StatusDialogDivider {
+                    Layout.fillWidth: true
+                    visible: assetsListView.count
+                }
+
+                ListView {
+                    id: assetsListView
+
+                    clip: true
+
+                    Layout.fillWidth: true
+                    Layout.fillHeight: true
+                    Layout.preferredHeight: contentHeight
+
+                    model: assetsSfpm
+
+                    delegate: TokenSelectorAssetDelegate {
+                            required property var model
+                            required property int index
+
+                        highlighted: tokensKey === d.currentTokensKey
+                        interactive: tokensKey !== root.nonInteractiveDelegateKey
```

Comment:
To selectively mark one token as non-interactive, in case you have 2 TokenSelectors side-by-side, as it's the case for SwapModal:
![image](https://github.com/status-im/status-desktop/assets/5377645/8e5a1177-7d40-4f7f-b5af-ac817da6e869)

"DAI" is disabled because it's selected in the other half already; this is purely a visual thing

Reply:
Ok, I got it, thanks. I will handle it when integrating to swap, probably along with some other adjustments here.